doc/howto/network_ovn_peers.md

```diff
@@ -5,34 +5,56 @@ discourse: 12165
 (network-ovn-peers)=
 # How to create peer routing relationships
 
-Network peers allow the creation of routing relationships between two OVN networks.
-This allows for traffic between those two networks to stay within the OVN subsystem rather than having to transit via the uplink network.
+By default, traffic between two OVN networks goes through the uplink network.
+This routing is inefficient, however, because it limits the network bandwidth to the uplink network's capabilities.
 
+Therefore, LXD allows creating peer routing relationships between two OVN networks.
+Using this method, traffic between the two networks can go directly from one OVN network to the other and thus stays withing the OVN subsystem, rather than transiting through the uplink network.
 
 ## Create a routing relationship between networks
 
-Both networks in the peering are required to complete a setup step to ensure that the peering is mutual.
+To add a peer routing relationship between two networks, you must create a network peering for both networks.
+Use the same peering name for both networks to ensure that the relationship is mutual.
```

Comment:
> Use the same peering name for both networks to ensure that the relationship is mutual.

This isn't true or required.

Reply:
Oh okay - so the name isn't really important at all? Just to identify the "rule" when editing it?

Reply:
Yes thats right. Its a friendly name to describe the peering relationship.